problems/objects.js

```diff
@@ -17,12 +19,21 @@ const eveAppleCount = () => {};
  * @param {Object} appleCountByName
  * @param {string} name - name of person
  * @returns {number} The number of apples name has.
- *
+ *`
  * Is there a difference between using string notation vs
- * bracket notation? Try them both.
+ * bracket notation? Try them both.`
  */
 
-const appleCount = () => {};
+const appleCount = (appleCountByName, name) => {
+  // let appleCountByName.name2 = name
+  // console.log(appleCountByName.name);
+  if (!appleCountByName[name]) {
+    return 0;
```

Comment:
Awesome! Yes, if someone's name isn't in the object it's a great idea to have the function return 0 apples than undefined apples!